internal/logging/interceptor.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+// Package logging is a copy of https://github.com/grpc-ecosystem/go-grpc-middleware/tree/v2/providers/zerolog
```

Comment:
Did you file an issue with them as well?

Reply:
https://github.com/grpc-ecosystem/go-grpc-middleware/pull/487

Also removed the mutex; made copies of the logger instead